frontend/src/index.js

```diff
@@ -1,55 +1,25 @@
+// @flow
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
-import {
-  createStore,
-  applyMiddleware,
-  compose,
-  combineReducers,
-} from 'redux';
-import { createLogger } from 'redux-logger'
-import {
-  reducer as asyncDataFetchReducer,
-  middleware as asyncDataFetchMiddleware,
-  storeKey as asyncDataFetchStoreKey,
-  actions as asyncDataFetchActions,
-} from '@hharnisc/async-data-fetch';
+import store from './store'
+import { actions as asyncDataFetchActions } from './store/async-data-fetch-middleware'
```

Comment:
Why not grab this from the imported package? Looks like a copy :smile:

Reply:
i use a transform option on all middlewares that return data now. Wanted to show it to you and get your thoughts on it